### DISCUSSION TITLE
01w pro with Gotek Please help

### DISCUSSION THREAD

**Discussion by arbodel** [keirf/flashfloppy#847] [General]
Gotek AT32F435 device with jumper S0 (sfrkc30.at4.35 ,  3.38 ff+) installed on 01w pro. Error 34 with usb inserted. Your ff.cng(modified- shugart, akai) , img.cng(unmodified) and 720k(2nd zip) copied on multiple good usb sticks(fat32)-still e34.
Any advise would be appreciated.Green light is always on.

**Comment by keirf** [comment]
E34 means no recognised image file. So your IMG file does not have IMG suffix, or is an unexpected size (eg. it's not exactly 720kb in size)

**Reply by arbodel** [reply]
I use your 720k 2nd zip you have for someone in issue #385

**Reply by arbodel** [reply]
By the way green light is always on...is that the way it should be?

**Reply by keirf** [reply]
For some systems light always on is correct.

**Reply by arbodel** [reply]
Ok , got it recognized by korg - (korg sees the img files on usb!)...but ...new problem in next step - formatting on korg ends in "data error"  on Korgs display (used few 720k img files from github).
Does IMG.CNG file configuration has anything to do with formatting error.If yes, is this the right one (this what i use) ...

[os::737280]
cyls = 80
heads = 2
secs = 9
bps = 512
tracks = 0-79 # This line can be adjusted
  interleave = 2
  hskew = 1
  cskew = 2

# Matches 720kB images named *.sys.{img,ima,dsk}.
[sys::737280]
cyls = 80
heads = 2
secs = 9
bps = 512
tracks = 0-79 # This line can be adjusted
  interleave = 2
  hskew = 1
  cskew = 2

**Comment by arbodel** [comment]
for At32F435 which soft version do you think is the best in terms of functionality

**Reply by keirf** [reply]
Any should be okay. Try removing IMG.CFG file and see how you get on.

Korg generally also uses Akai style 800kB disk format. So it is probbaly trying to format as 800kB and that's incompatible with a 720kB IMG file. So make a800kB IMG file instead (it must be 800kB = 819200 bytes exactly in length) and try formatting that.

Or as a fallback, you can work with double-density HFE image files. There is a blank one here: https://github.com/keirf/flashfloppy-images/blob/master/Unformatted/HFE/double_density.hfe (download raw)

**Reply by arbodel** [reply]
Ill try.TU

**Reply by arbodel** [reply]
... how about 1mb, whats the exact length of 1 mb-( I read somewhere that korg makes 720kb out of 1mb)
1024000 got it

**Reply by arbodel** [reply]
800kb disk formatting-data error

**Reply by keirf** [reply]
Is the Gotek jumper at S0 only? Have you tried flipping the data connector in case it is wrong way round?

**Comment by arbodel** [comment]
if you could provide 1mb IMG to try please.

**Reply by keirf** [reply]
Try the double density HFE image file that I linked to in a previous reply.

**Reply by arbodel** [reply]
ok

**Reply by arbodel** [reply]
put dd hfe and ff.cfg - deta error

**Reply by arbodel** [reply]
34-pin Floppy Drive Pins for 01w
Pin	Name	Direction	Description
2	unused 
REDWC-	- 
to drive	in original PC
Density Select (in PC-AT?)
4	unused	-	not connected
6	unused	-	not connected
8	INDEX-	from drive	Index (one pulse each rotation)
10	MOTEA-	to drive	Motor Enable drive-A
12	DRVSB-	to drive	Drive Select drive-B
14	DRVSA-	to drive	Drive Select drive-A
16	MOTEB-	to drive	Motor Enable drive-B
18	DIR-	to drive	Stepper motor direction (0=towards spindle)
20	STEP-	to drive	Step pulse
22	WDATE-	to drive	Write Data
24	WGATE-	to drive	Write Enable
26	TRK00-	from drive	Track 0 (0=track 0)
28	WPT-	from drive	Write Protect (0=protected)
30	RDATA-	from drive	Read Data (pulse=flux change)
32	SIDE1-	to drive	Select Head 1(0=top, 1=bottom)
34	unused
Ready(?)
DSKCHG-?	-
?
to drive?	in original PC (this I know)
Drive Ready? (in PC-XT?)
Disk Change (in PC-AT?)

**Comment by arbodel** [comment]
nothing works...

**Comment by arbodel** [comment]
please help
